Makefile

```
@@ -0,0 +1,35 @@
+.PHONY: compile
+compile:
+	rebar3 compile
+
+.PHONY: test
+test:
+	rebar3 ct
+
+.PHONY: release
+release:
+	rebar3 as release escriptize
+
+.PHONY: clean
+clean:
+	rm -rf _build
+	rebar3 clean
+
+.PHONY: fmt
+fmt:
+	make clean
+	make release
+	_build/release/bin/erlfmt -w "{src,include,test}/*.{hrl,erl,app.src}" "rebar.config"
```

Any reason for not using target dependencies?
```suggestion
fmt: clean release
	_build/release/bin/erlfmt -w "{src,include,test}/*.{hrl,erl,app.src}" "rebar.config"
```

I'm also not sure we do need to rebuild everything every time we compile. Any particular reason for this?

---

Paranoia and Parity with CI
But I have changed it, so that only the `check` command does `clean` now.

Generally I prefer calling make over target dependencies, for readability, but I have now added target dependencies to conform :)